src/index.html

```diff
@@ -1,16 +1,58 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <div class="search_styles">
+        <label class="big_search_bar_one">
+          <input type="text" 
+                 data-qa="keypress" 
+                 name="bigSearchBar" 
+                 class="input_bar big_search_bar"
+                 placeholder="Try &quot;Los Angeles&quot;"
+          >
```

Comment:
```suggestion
          <input 
             type="text" 
             data-qa="keypress" 
             name="bigSearchBar" 
             class="input_bar big_search_bar"
             placeholder="Try &quot;Los Angeles&quot;"
           >
```

Reply:
when you have > 2 attributes move every attribute to new line with indentation +2
closing bracket should be on new lane on the same lvl with opening bracket

Reply:
![image](https://user-images.githubusercontent.com/55200530/222150853-34a3f797-740a-4901-a1f2-9a1eee1b311f.png)